bitcoin/examples/ecdsa-psbt.rs

```diff
@@ -130,7 +130,9 @@ impl ColdStorage {
     }
 
     /// Returns the fingerprint for the master extended public key.
-    fn master_fingerprint(&self) -> Fingerprint { self.master_xpub.fingerprint() }
+    fn master_fingerprint(&self) -> Fingerprint {
+        self.master_xpub.fingerprint()
+    }
```

Comment:
This is an unrelated (incorrect) formatting change. Have you got your editor set to format on save but using the stable toolchain?

Reply:
There's a bunch of other places, will review after those are gone. Sorry, `rustfmt` presents a bunch of basically unsolvable annoyances, one of which is it has to be run in this codebase with `nightly`.

Reply:
They are gone, thanks for pointing them out.